downstream/modules/platform/ref-platform-components.adoc

```diff
@@ -0,0 +1,50 @@
+ 
+.{PrivateHubNameStart}
+If you create your own Ansible content, the Ansible private automation hub provides a way to publish and collaborate on it within your organization behind your firewall or on disconnected systems. 
+You can also add to it curated content from {HubNameMain} and {Galaxy}.
+
+ 
+.{ControllerNameStart}
+An enterprise framework for controlling, securing, and managing Ansible automation with a user interface (UI) and RESTful application programming interface (API).
+ 
+.{CatalogNameStart}
+{CatalogNameStart} is a service within {PlatformName}. 
+{CatalogNameStart} enables you to organize and govern product catalog sources on Ansible {ControllerName} across various environments.
+
+Using automation services catalog you can:
+
+* Apply multi-level approval to individual platform inventories.
+* Organize content in the form of products from your platforms into portfolios.
+* Choose portfolios to share with specific groups of users.
+* Set boundaries around values driving execution of user requests.
+ 
+.{AutomationMeshStart}
+{AutomationMeshStart} is an overlay network intended to ease the distribution of work across a large and dispersed collection of workers through nodes that establish peer-to-peer connections with each other using existing networks.
+
+{AutomationMeshStart} provides:
+
+* Dynamic cluster capacity that scales independently, allowing you to create, register, group, ungroup and deregister nodes with minimal downtime.
+* Control and execution plane separation that enables you to scale playbook execution capacity independently from control plane capacity.
+* Deployment choices that are resilient to latency, reconfigurable without outage, and that dynamically re-reroute to choose a different path when outages may exist. mesh routing changes.
+* Connectivity that includes bi-directional, multi-hopped mesh communication possibilities which are Federal Information Processing Standards (FIPS) compliant.
+
+.{ExecEnvNameStart}
+A solution that includes the Ansible execution engine and hundreds of modules that help users automate all aspects of IT environments and processes. 
+Execution environments automate commonly used operating systems, infrastructure platforms, network devices, and clouds. 
+ 
+.{Galaxy}
+A hub for finding, reusing, and sharing Ansible content. 
+Community-provided Galaxy content, in the form of prepackaged roles, can help start automation projects. Roles for provisioning infrastructure, deploying applications, and completing other tasks can be dropped into Ansible Playbooks and be applied immediately to customer environments. 
+ 
+.{Navigator}
+A _textual user interface_ (TUI) that becomes the primary command line interface into the automation platform, covering use cases from content building, running automation locally in an execution environment, running automation in {PlatformNameShort}, and providing the foundation for future _integrated development environment_ (IDEs).
```

Comment:
Suggestion to unitalicize the spelled-out abbreviations to remain consistent w/other spelled-out abbreviations. I took a look at the [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations#spelled-out-forms) to see what it recommends, but it doesn't specifically mention italicizing spelled-out abbreviations. Although, from the examples, I don't see any spell-out abbreviations italicized. 

I'll leave this decision up to you.

Reply:
I thought this was standard actually. First use of name italicize the name and give the acronym in brackets. It seemed to be used on Satellite and in most places I've worked.

Reply:
I reviewed the AAP guides and did not see spelled out acronyms to be italicized. I also don't see any guidance for italicizing or not italicizing them so I leave this change up to you. Though, this is something I'd like to look into to keep our documentation consistent throughout CCS Docs.